README.md

```diff
@@ -1,5 +1,37 @@
 # techmiya-commercials
 
-## TODO
+## How to setup
 
-Add Setup/Usage details
+Follow this guide to setup this project on your local machine.
+
+1. Install [python] 2.x, git and [virtualenv] in your computer.
+
+2. Get the source code on your machine by-
+
+    `git clone https://github.com/sitture/techmiya-commercials.git`
+
+3. Create a python virtual environment and install python and django related dependencies.
+
+    ```shell
+    cd website
```

Comment:
should this be `cd techmiya-commercials`?

Reply:
@haroon-sheikh Oh. Doing that